src/app.js

```diff
@@ -1,21 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+
+  const normalizedURL = new url.URL(req.url, `http://${req.headers.host}`);
+  const parts = normalizedURL.pathname.slice(1).split('/');
+
+  if (parts[0] !== 'file') {
+    res.end('Pathname must start with "/file/"');
```

Comment:
you don't need double quotes here
```suggestion
    res.end('Pathname must start with /file/');
```